virtool/startup.py

```diff
@@ -262,7 +267,7 @@ async def startup_databases(app: Application):
             "db": DB(mongo, dispatcher_interface.enqueue_change, RandomIdProvider()),
             "dispatcher_interface": dispatcher_interface,
             "pg": pg,
-            "auth": AuthorizationClient(mongo, openfga_instance),
+            "authorization": AuthorizationClient(mongo, openfga_instance),
```

Comment:
```suggestion
            "authorization": AuthorizationClient(openfga_instance),
```
Mongo is no longer used in the Authorization client.